src/ServiceInsight/Framework/Settings/ApplicationConfiguration.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace ServiceInsight.Framework.Settings
+{
+    using System;
+    using System.Configuration;
+
+    public static class ApplicationConfiguration
+    {
+        public static bool SkipCertificateValidation { get; private set; }
+
+        public static void Initialize()
+        {
+            SkipCertificateValidation = GetSkipCertificateValidation();
+        }
+
+        private static bool GetSkipCertificateValidation()
+        {
+            try
+            {
+                return Convert.ToBoolean(ConfigurationManager.AppSettings["SkipCertificateValidation"] ?? "false");
```

Comment:
I looked into `Convert.ToBoolean()` and it turns out that it already converts `null` to `false`:

![image](https://user-images.githubusercontent.com/677704/32484455-c902d40c-c397-11e7-9ff7-78b50e45f62a.png)

Indeed, this is the [only difference between `Convert.ToBoolean()` and `bool.Parse()`](https://stackoverflow.com/a/7032036/49241).

This makes `?? "false"` redundant. Shall we remove it?

Reply:
The only bug free code is deleted code. :+1: to remove

Reply:
Rolled back!